### DISCUSSION TITLE
Every programming language has its "bible" - now DELPHI 12 has its own!

### DISCUSSION THREAD

**Discussion by hindermath** [holgerflick/flixaccounting#30] [General]
Every programming language has its "bible". C e.g. https://www.amazon.com/-/de/dp/0131103628/ 
or TurboPascal e.g. https://www.amazon.de/Turbo-Pascal-7-0-Rudolf-Herschel/dp/3486230166/.
Modern DELPHI programming with desktop, database/ORM and REST API components now has its own "bible":
Modern Application Development with Delphi (Color Print): Learn to develop a desktop application, plus an API and an ORM from @holgerflick 
https://www.amazon.com/-/de/dp/B0CQMHMX3F/

**Comment by holgerflick** [comment]
Thank you for the kind words. I posted your feedback to share with the community.

**Reply by hindermath** [reply]
Have seen it on LinkedIn.
